modules/infra/control/port.c

```diff
@@ -307,6 +307,9 @@ static int iface_port_fini(struct iface *iface) {
 		LOG(ERR, "rte_eth_dev_info_get: %s", rte_strerror(-ret));
 	if ((ret = rte_eth_dev_stop(port->port_id)) < 0)
 		LOG(ERR, "rte_eth_dev_stop: %s", rte_strerror(-ret));
+	/* DPDK limitation */
```

Comment:
The issue is related to the driver/bus, not to DPDK itself. Possibly copy pasting the commit message. Also, please use C++ style line comments:

```c
// foo bar baz
```